etc/setup/templates/configs/keycloak/configuration/secrets.env.tpl

```diff
@@ -0,0 +1,3 @@
+KEYCLOAK_USER=${KC_ADMIN_USER}
+KEYCLOAK_PASSWORD=${KC_ADMIN_PW}
+###PROXY_ADDRESS_FORWARDING=${PROXY_ADDRESS_FORWARDING}
```

Comment:
What is this for?  Is this needed?

Reply:
Yes, Keycloak uses it for its admin credentials at  boot time